src/sas/qtgui/Plotting/Plotter.py

```diff
@@ -301,16 +282,6 @@ def plot(self, data=None, color=None, marker=None, hide_error=False, transform=T
         # refresh canvas
         self.canvas.draw_idle()
 
-    def onResize(self, event):
```

Comment:
I am not entirely sure if we completely should get rid of it. Could there is still be the case e.g. on low resolutions screen that this may be useful

Reply:
The underlying _legendResize is gone in this, originally done by @krzywon.  The gist is that matplotlib now provides its own handler for this and we don't need to duplicate that functionality any more.